### PR TITLE
Fix function redeploy

### PR DESCRIPTION
**Pull request by jimcadden**

### CODE REVIEW

**Comment by jimcadden** [comment]
@tommy-u  I just did a quick scan of the diff: Before we merge, please clean up the comment-out code in `UmSV.cc` and `UmRegion.cc` and also confirm that all the tests (AppMain) correctly conform to the updated `UmManager` interfaces.

File: src/UmLoader.cc [hunk 1]
```diff
@@ -144,5 +144,22 @@ umm::ElfLoader::createSVFromElf(unsigned char *elf_start) {
   usr_reg.writable = true;
   ret_state.AddRegion(usr_reg);
 
+  // HACK: With the ef zeroed in the sv constructor, this is the minimal state
```
Comment (jimcadden) [inline src/UmLoader.cc]: Should this hack be instead in the default constructor of the SV?

File: src/UmManager.cc [hunk 2]
```diff
@@ -308,13 +318,13 @@ void umm::UmManager::Halt() {
...
 }
 
-void umm::UmManager::Kickoff() {
-  kassert(status() == loaded);
-    kprintf_force(GREEN "\nUmm... Kicking off the instance on core #%d\n" RESET,
-                  (size_t)ebbrt::Cpu::GetMine());
-  set_status(kickoff);
-  trigger_bp_exception();
-}
+// void umm::UmManager::Kickoff() {
```
Comment (jimcadden) [inline src/UmManager.cc]: Delete old code

File: src/UmManager.cc [hunk 3]
```diff
@@ -70,44 +70,58 @@ void debugKickoff(ebbrt::idt::ExceptionFrame *ef){
...
+// void printEF(ebbrt::idt::ExceptionFrame *ef){
+//   uint64_t *regPtr = (uint64_t *)ef;
+//   for (unsigned int i = 0; i < sizeof(ebbrt::idt::ExceptionFrame) / 8; i++, regPtr++) {
+//     kprintf("[%d] %p %d\n", i, *regPtr, 8*i);
+//   }
+//   // while(1);
+// }
 
-    // debugKickoff(ef);
+// void minimize_current_ef(ebbrt::idt::ExceptionFrame *ef){
```
Comment (jimcadden) [inline src/UmManager.cc]: Delete test code

**Review by jimcadden — COMMENTED** [review]
New additions look good. Just clean up the few code artifacts and it upto you where the hack should be

**Comment by tommy-u** [comment]
Addressed @jmcadden notes, rebasing.

**Pull request merged**